website/src/components/Changelog/Apple.tsx

```diff
@@ -24,7 +24,13 @@ export default function Apple() {
   return (
     <Entries downloadLinks={downloadLinks} title="macOS / iOS">
       {/* When you cut a release, remove any solved issues from the "known issues" lists over in `client-apps`. This must not be done when the issue's PR merges. */}
-      <Unreleased></Unreleased>
+      <Unreleased>
+        <ChangeItem pull="10056">
+          Fixes an issue where Firezone's system resolvers may not be when the
+          system's DNS resolvers change unless there was also a network
+          connectivity change.
```

Comment:
Not quite English! :)


```suggestion
          Fixes an issue where Firezone's DNS resolvers would not update when the
          system's DNS resolvers change unless there was also a network
          connectivity change.
```